java/ingest/src/test/java/sleeper/ingest/testutils/IngestRecordsTestDataHelper.java

```diff
@@ -51,18 +51,19 @@ private IngestRecordsTestDataHelper() {
 
 
     public static IngestProperties.Builder defaultPropertiesBuilder(
-            StateStore stateStore, Schema sleeperSchema, String tempDirectory) {
+            StateStore stateStore, Schema sleeperSchema, String ingestLocalWorkingDirectory, String sketchDirectory) {
         return IngestProperties.builder()
                 .objectFactory(ObjectFactory.noUserJars())
-                .localDir(tempDirectory)
+                .localDir(ingestLocalWorkingDirectory)
                 .maxRecordsToWriteLocally(10L)
                 .maxInMemoryBatchSize(1000L)
                 .rowGroupSize(ParquetWriter.DEFAULT_BLOCK_SIZE)
                 .pageSize(ParquetWriter.DEFAULT_PAGE_SIZE)
                 .compressionCodec("zstd")
                 .stateStore(stateStore)
                 .schema(sleeperSchema)
-                .filePathPrefix(tempDirectory)
+                .filePathPrefix(sketchDirectory)
+                .bucketName(sketchDirectory)
```

Comment:
These two properties will get joined together to make the actual directory used. We shouldn't need to set the bucket name. There might be a better way to set up the properties.